nuxeo-retention/src/main/resources/OSGI-INF/l10n/messages_fr.properties

```diff
@@ -0,0 +1,4 @@
+label.directories.retention_end.delete=Supprimer
```

Comment:
I think other locale than english are handled by crowdin, see https://doc.nuxeo.com/nxdoc/how-to-translate-the-nuxeo-platform/#nuxeo-crowdin-synchronization

Reply:
forget it. There is no crowdin integration for the retention addon.

Reply:
We do have crowdin for retention, for example: https://github.com/nuxeo/nuxeo-retention/commit/d6da43795092d563872ea3e83fea2454e6e9f8e0